docs/bump_version.md

```diff
@@ -0,0 +1,45 @@
+# How to bump a package version for an existing recipe in ConanCenterIndex?
```

Comment:
I don't think this is what the complaints are about...

I think people assume that updating a package (which is naively new versions) is different from adding a package.

I absolutely agree there's a problem but this seem more narrow than what is needed 🤔 

I would suggest a new doc "update existing recipes" and i would image a "adding new versions"  along with links to the other more details docs tie them together 

Just me thinking out loud

Reply:
> I would suggest a new doc "update existing recipes" and i would image a "adding new versions" along with links to the other more details docs tie them together

This would fit yes! But I would prefer step-by-step. Documentation is always massive to review, so small PRs are easier to digest.